### DISCUSSION TITLE
WebDriverRunner.closeWebDriver() issue

### DISCUSSION THREAD

**Discussion by AntonTrigubovich** [selenide/selenide#2736] [Q&A] [answered]
We using latest version of selenide and chrome(Version 124.0.6367.119 (Official Build) (64-bit)). We use WebDriverRunner.closeWebDriver() method after every test. After running our test, the Chrome process remains unclosed. Chromedrive.exe closed, but chrome process not. 
![image](https://github.com/selenide/selenide/assets/87859634/2fd5653c-ac34-4b2b-b298-c30a84863437)
I can close it only in task manager. It takes a lot of CPU, 3 of these processes take all CPU. 
I started researching this issue and I can say that if we use WebDriverRunner.closeWebDriver() or WebDriverRunner.getWebDriver().quit();  we are facing this problem, also WebDriverRunner.hasWebDriverStarted() return true. But using WebDriverRunner.getWebDriver().close(); instead of WebDriverRunner.closeWebDriver() the problem disappears. If I understand correctly WebDriverRunner.closeWebDriver() using quit() method inside. So it's more of a selenium problem. But anyway do someone faceting this issue?

**Comment by BorisOsipov** [comment]
>But anyway do someone faceting this issue?

https://issues.chromium.org/issues/336164423

**Reply by AntonTrigubovich** [reply, marked as answer]
thanks a lot, removing the chrome "--no-sandbox" option solved the problem.
